### PR TITLE
[zh]Sync content/zh/docs/tasks/security/authentication/mtls-migration/index.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by windsonsea** [comment]
Please resolve the lint error before reviewing.

File: content/zh/docs/tasks/security/authentication/mtls-migration/index.md [hunk 1]
```diff
@@ -11,24 +11,27 @@ test: yes
...
-        * `bar`
+* 有一个安装了 Istio 的 Kubernetes 集群，但没有启用全局双向 TLS（例如，使用[安装步骤](/zh/docs/setup/getting-started)中描述的 `default` 配置文件）。
 
-    * 创建如下命名空间并在其中部署 [sleep]({{< github_tree >}}/samples/sleep)，不注入 sidecar
-        * `legacy`
+在此任务中，您可以通过创建示例工作负载并修改策略以在工作负载之间强制执行 STRICT 双向 TLS 来尝试迁移过程。
+
+## 设置集群{#set-up-the-cluster}
+
+* 创建两个命名空间，`foo` 和 `bar`，并它们上部署 [httpbin]({{< github_tree >}}/samples/httpbin)、[sleep]({{< github_tree >}}/samples/sleep) 和 sidecar。
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/authentication/mtls-migration/index.md:34]:
```suggestion
* 创建两个命名空间：`foo` 和 `bar`，并在这两个命名空间上部署 [httpbin]({{< github_tree >}}/samples/httpbin)、[sleep]({{< github_tree >}}/samples/sleep) 和 Sidecar。
```

File: content/zh/docs/tasks/security/authentication/mtls-migration/index.md [hunk 2]
```diff
@@ -37,75 +40,51 @@ test: yes
     $ kubectl create ns bar
     $ kubectl apply -f <(istioctl kube-inject -f @samples/httpbin/httpbin.yaml@) -n bar
     $ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@) -n bar
+    {{< /text >}}
+
+* 创建另一个命名空间 `legacy`，并在没有 sidecar 的情况下部署 [sleep]({{< github_tree >}}/samples/sleep)：
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/authentication/mtls-migration/index.md:45]:
```suggestion
* 创建另一个命名空间 `legacy`，并在没有 Sidecar 的情况下部署 [sleep]({{< github_tree >}}/samples/sleep)：
```

File: content/zh/docs/tasks/security/authentication/mtls-migration/index.md [hunk 3]
```diff
@@ -37,75 +40,51 @@ test: yes
...
+
+    {{< text bash >}}
     $ kubectl create ns legacy
     $ kubectl apply -f @samples/sleep/sleep.yaml@ -n legacy
     {{< /text >}}
 
-* （使用 curl 命令）从每个 sleep pod （命名空间为 `foo`，`bar` 或 `legacy`）分别向 `httpbin.foo` 发送 http 请求。所有请求都应成功响应，返回 HTTP code 200。
+* （使用 Curl 命令）从每个 sleep pod （命名空间为 `foo`，`bar` 或 `legacy`）分别向 `httpbin.foo` 发送 http 请求。所有请求都应成功响应，返回 HTTP code 200。
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/authentication/mtls-migration/index.md:52]:
```suggestion
* （使用 curl 命令）从每个 Sleep Pod （命名空间为 `foo`、`bar` 或 `legacy`）分别向 `httpbin.foo` 发送 http 请求。所有请求都应成功响应，返回 HTTP code 200。
```

File: content/zh/docs/tasks/security/authentication/mtls-migration/index.md [hunk 4]
```diff
@@ -37,75 +40,51 @@ test: yes
...
     sleep.bar to httpbin.foo: 200
+    sleep.bar to httpbin.bar: 200
     sleep.legacy to httpbin.foo: 200
+    sleep.legacy to httpbin.bar: 200
     {{< /text >}}
 
-* 验证没有在系统中设置认证策略或目标规则（控制面板除外）：
+    {{< tip >}}
+
+如果任何 Curl 命令失败，请确保可能干扰 httpbin 服务请求的现有身份验证策略或目标规则。
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/authentication/mtls-migration/index.md:66]:
```suggestion
如果任何 curl 命令失败，请确保可能干扰 httpbin 服务请求的现有身份验证策略或目标规则。
```

File: content/zh/docs/tasks/security/authentication/mtls-migration/index.md [hunk 5]
```diff
@@ -162,9 +141,15 @@ EOF
...
 {{< /text >}}
 
-## 清除{#cleanup}
+## 清除{#clean-up-the-example}
+
+1. 删除网格范围的身份验证策略。
+
+{{< text bash >}}
+$ kubectl delete peerauthentication -n istio-system default
+{{< /text >}}
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/authentication/mtls-migration/index.md:150]:
```suggestion
    {{< text bash >}}
    $ kubectl delete peerauthentication -n istio-system default
    {{< /text >}}
```

File: content/zh/docs/tasks/security/authentication/mtls-migration/index.md [hunk 6]
```diff
@@ -162,9 +141,15 @@ EOF
...
 
-清除所有资源。
+1. 删除用于测试的命名空间。
 
 {{< text bash >}}
 $ kubectl delete ns foo bar legacy
```
Comment (windsonsea) [inline content/zh/docs/tasks/security/authentication/mtls-migration/index.md:155]: The four rows need 4 spaces indentation.

**Review by windsonsea — CHANGES_REQUESTED** [review]

File: content/zh/docs/tasks/security/authentication/mtls-migration/index.md (hunk 4 above)
Comment (yanrongshi) [reply content/zh/docs/tasks/security/authentication/mtls-migration/index.md:66]: 或许应该统一成大写？

Comment (windsonsea) [reply content/zh/docs/tasks/security/authentication/mtls-migration/index.md:66]: 搜索了一下 k8s 和 istio repo，都是纯小写 `curl`

**Review by windsonsea — APPROVED** [review]
Looks good to me, thanks.

**Pull request merged**